django/contrib/admin/templates/admin/change_list_results.html

```diff
@@ -10,7 +10,7 @@
 <thead>
 <tr>
 {% for header in result_headers %}
-<th scope="col" {{ header.class_attrib }}>
+<th scope="col" {{ header.class_attrib }} {% if header.help_text %}title="{{ header.help_text }}"{% endif %}>
```

Comment:
I prefer to put space inside condition, to eliminate useless space on false condition